Makefile

```diff
@@ -54,6 +54,7 @@ PASSWORD_STORE_DIR?=${HOME}/.paas-pass
 globals:
 	$(eval export AWS_DEFAULT_REGION=eu-west-1)
 	$(eval export PASSWORD_STORE_DIR=${PASSWORD_STORE_DIR})
```

Comment:
In the original implementation we use a different variable for datadog, I did not change that part of the code. In that sense, the "interface" with the rest of the project (e.g. environment variables) is the same. We can trace the code back to the original PR.